handbook/company/pricing-features-table.yml

```diff
@@ -1160,4 +1123,4 @@
   jamfProtectHasFeature: yes
   usualDepartment: IT
   productCategories: [Device management]
-  pricingTableCategories: [Device management]
+  pricingTableCategories: [Integrations]
```

Comment:
Moved "Munki compatibility" to integrations because it's not an MDM feature per se, especially now that we have shipped the app deployment feature.